### PR TITLE
Add IRPF 2022

### PR DESCRIPTION
**Pull request by guihkx**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [ ] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

---

**Disclaimer: I am not affiliated with the Brazilian government in any way.**

**App description:** This app is developed and distributed by the Brazilian government to assist citizens in filing their annual tax information. They release a major version of this app annually (usually on March), and this is the 2022 version, released last year, but still maintained. 

**App website (in Portuguese):** https://www.gov.br/receitafederal/pt-br/centrais-de-conteudo/download/pgd/dirpf

### CODE REVIEW

File: patches/serpro/ppgd/irpf/gui/NavegadorHtml.class
Comment (guihkx) [inline patches/serpro/ppgd/irpf/gui/NavegadorHtml.class:1]: Included this one by mistake. Will remove it.

**Comment by guihkx** [comment]
bot, build br.gov.fazenda.receita.irpf2022

**Comment by orowith2os** [comment]
Why not use one repo for this app, rather than resubmit for each new release? You don't see other apps doing this, why should IRPF?

**Comment by orowith2os** [comment]
Looking at https://github.com/flathub/flathub/pull/4095, it's because you need the same major version of the app to edit info from that version. Why don't later versions of the app work with info from earlier versions? I also feel like flathub setting up some system for branches other than beta and stable would be nice to have here.

**Comment by barthalion** [comment]
These comments under a submission I have already reviewed and approved rub me the wrong way, but alas.

> Why don't later versions of the app work with info from earlier versions?

Because they just don't, I'm not sure what answer are you expecting to that? 

> I also feel like flathub setting up some system for branches other than beta and stable would be nice to have here.

Branches deliver poor UX, go and try it yourself without using a terminal.

**Comment by guihkx** [comment]
>Why don't later versions of the app work with info from earlier versions?

I'm afraid that's a question that can only be answered by the Brazilian government, or by the people they hired to develop the app. :)

On their website, you can download versions of the app released 5 years prior from the current release, and they're all still maintained.

**Pull request closed without merging**